web/src/main/resources/templates/error/400.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+  <head th:replace="fragments/head"></head>
+
+  <body class="bg-dark bg-opacity-10">
```

Comment:
```suggestion
  <body>
```

I removed all the grey backgrounds in the UI update so just have to change them back to white.